test/macros.jl

```diff
@@ -801,6 +802,19 @@ end
             @NLparameter(m, p[m = 1:2] == m),
         )
     end
+
+    @testset "Broadcasting sparse arrays" begin
+        model = Model()
+        A = sparse([1, 2], [1, 2], [1, 1])
+        b = sparsevec([1, 2], [1, 2])
+        @variable(model, x[1:2])
+        le_cons = @constraint(model, A * x .<= b)
```

Comment:
In this examples, the sparse vectors are dense, can you add an example with structural zero elements ?